cluster/images/etcd/migrate/options.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 var (
-	supportedEtcdVersions = []string{"3.4.18", "3.5.21", "3.6.0-rc.2"}
+	supportedEtcdVersions = []string{"3.4.18", "3.5.21"}
```

Comment:
I am not sure whether we should remove `3.6.0-rc.2` (We will bump 3.6.0-rc.3 regardless after release-1.33 branch is cut). I guess we won't be able to run the upgrade test anymore until we bump 3.6.0-rc.x again. cc @neolit123 @serathius

Reply:
/cluster dir is still used for testing, but doesn't matter much from my pov.

Reply:
thx @neolit123 for the comment. Not sure I got your point. Do you mean that you can still trigger the upgrade (etcd 3.5 -> 3.6) e2e test as you mentioned in [slack message](https://kubernetes.slack.com/archives/C3HD8ARJ5/p1743168503811089?thread_ts=1743165381.728499&cid=C3HD8ARJ5) even we remove `3.6.0-rc.2` here and in [cluster/images/etcd/Makefile](https://github.com/kubernetes/kubernetes/pull/131126/files/0e2e687d3cc8e7e7537bf4a5465cb8d834f36848#diff-de315dbb88d6d341b1682ef2fb0798723139fb4202ac607a334fc301e9359ec7) ?

Reply:
cluster dir is unrelated to kubeadm.
we should clean all 3.6.0 bits from 1.33 (master branch)

Reply:
merging etcd rc changes should be done in the beginning of a k8s release cycle, presumably meaning that by the end of the release the non-rc version will merge too. but again, the cluster dir doesn't matter much and is used for testing only.

Reply:
OK, thx

Reply:
pls mark this comment as resolved, thx